examples/source/1.components/amp-inputmask.html

```diff
@@ -44,7 +44,7 @@
 
      In the mask, `L` allows the user to enter an alphabetic character. `0` denotes a numeric character. `_` is a literal space character. See the full mask specification at [the `amp-inputmask` documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-inputmask/README.md)
    -->
-  <form class="sample-form" method="post" action-xhr="<% hosts.backend %>/components/amp-inputmask/postal" target="_top">
+  <form class="sample-form" method="post" action-xhr="/documentation/examples/api/postal" target="_top">
```

Comment:
To be consistent I'd find it good practice to keep using absolute URLs here and prepend with `<% hosts.platform %>`.